perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/MembersManagerMethod.java

```diff
@@ -390,6 +390,19 @@ public Member call(ApiCaller ac, Deserializer parms) throws PerunException {
  	 * @param lookingInParentGroup boolean If true, look up in a parent group
  	 * @return List<RichMember> List of richMembers with specific attributes from Group
  	 */
+	/*#
+	 * Get all RichMembers with attributes specific for list of attrNames.
+	 * Attributes are defined by member (user) and resource (facility) objects.
+	 * It returns also user-facility (to userAttributes of RichMember) and
+	 * member-resource (to memberAttributes of RichMember) attributes.
+	 * Members are defined by group and are filtered by list of allowed statuses.
+	 *
+	 * @param group int Group <code>id</code>
+	 * @param resource int Resource <code>id</code>
+	 * @param attrsNames List<String> Attribute names
+	 * @param allowedStatuses List<String> Allowed statuses (VALID | INVALID | SUSPENDED | EXPIRED | DISABLED)
+	 * @return list of richMembers with selected specific attributes
```

Comment:
``@return List<RichMember> list of richMembers....``, since RPC javadoc doesn't know return type.